src/appmixer/ai/gemini/AIAgent/AIAgent.js

```
@@ -0,0 +1,128 @@
+        if (threadId) {
+            await context.stateSet(`history:${threadId}`, messages);
+        }
+
+        while (true) {
+
+            await context.log({ step: 'turn', messages });
+
+            const result = await client.generateContent({ contents: messages });
+
+            const functionCalls = result.response.functionCalls();
+            if (functionCalls && functionCalls.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)